geomesa-features/geomesa-feature-common/src/main/scala/org/locationtech/geomesa/features/ScalaSimpleFeature.scala

```diff
@@ -155,10 +159,13 @@ class ScalaSimpleFeature(initialId: String, sft: SimpleFeatureType, initialValue
 
 object ScalaSimpleFeature {
 
+  def create(sft: SimpleFeatureType, copy: SimpleFeature): ScalaSimpleFeature =
+    new ScalaSimpleFeature(copy.getID, sft, copy.getAttributes.toArray, copy.getUserData)
+
   /**
    * Creates a simple feature, converting the values to the appropriate type
    */
-  def create(sft: SimpleFeatureType, id: String, values: Any*): SimpleFeature = {
+  def create(sft: SimpleFeatureType, id: String, values: Any*): ScalaSimpleFeature = {
```

Comment:
the syntax might get a little messy since it uses varargs... since it's only used in testing i'd be inclined to skip for now